### PR TITLE
Update references to the archived slack room #support-client-plat

### PR DESCRIPTION
**Pull request by sourcegraph-wk**
This batch changes references to the archived support-client-plat slack channel to now reference the support-ui-platform channel.

`replace "#support-client-plat" "#support-ui-platform"`

### QA steps:

CI passing should be sufficient as these changes are documentation changes.

[_Created by Sourcegraph batch change `Workiva/update_slack_channel_fed`._](https://sourcegraph.plat.workiva.net/organizations/Workiva/batch-changes/update_slack_channel_fed)

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: skynet.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: private-unit-tests
 description: Unit tests that require access to our private Pub server.
-contact: 'CPLAT / #support-client-plat'
+contact: 'CPLAT / #support-ui-platform'
```
Comment (kimlarson-wk) [inline skynet.yaml:3]:
```suggestion
contact: 'Frontend Design / #support-ui-platform'
```

**Review by greglittlefield-wf — APPROVED** [review]
+10

**Comment by greglittlefield-wf** [comment]
It looks like CI is just failing to report the tests for some reason. We can just ignore that failure and merge anyways

@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**